From stable-bounces@linux.kernel.org  Tue Feb  7 03:38:03 2006
Date: Tue, 07 Feb 2006 11:33:51 +0000
From: Daniel Drake <dsd@gentoo.org>
To: stable@kernel.org
Cc: tiwai@suse.de
Subject: [ALSA] emu10k1 - Fix the confliction of 'Front' control

From: Takashi Iwai <tiwai@suse.de>

Modules: EMU10K1/EMU10K2 driver

Fix the confliction of 'Front' controls on models with STAC9758 codec.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Chris Wright <chrisw@sous-sol.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 sound/pci/emu10k1/emumixer.c |    2 ++
 1 files changed, 2 insertions(+)

Index: linux-2.6.15.3/sound/pci/emu10k1/emumixer.c
===================================================================
--- linux-2.6.15.3.orig/sound/pci/emu10k1/emumixer.c
+++ linux-2.6.15.3/sound/pci/emu10k1/emumixer.c
@@ -750,6 +750,8 @@ int __devinit snd_emu10k1_mixer(emu10k1_
 		"Master Mono Playback Volume",
 		"PCM Out Path & Mute",
 		"Mono Output Select",
+		"Front Playback Switch",
+		"Front Playback Volume",
 		"Surround Playback Switch",
 		"Surround Playback Volume",
 		"Center Playback Switch",
